crates/biome_css_parser/src/lexer/mod.rs

```diff
@@ -1315,6 +1317,27 @@ impl<'src> CssLexer<'src> {
             _ => false,
         }
     }
+
+    fn re_lex_grit_metavariable(&mut self, current_end: usize) -> CssSyntaxKind {
+        if self.current_kind == T![ident]
+            && self.current_byte() == Some(b'$')
```

Comment:
I think we should change it to use the μ character. It’s what we’ll use for other languages as well.